forum/qa-content/qa-page.js

```diff
@@ -836,7 +837,16 @@ function qa_ajax_error()
                     idNumber = idNumber.slice( 1, idNumber.indexOf( '_' ) );
 
                     usersResponsesList = topicMainContent.querySelector( '[id*="' + idNumber + '_list"]' );
-                }
+                } else if ( areCommentsExpanded ) {
+                	var target = ev.target.parentNode.parentNode;
+                    var mutationObserver = new MutationObserver( function() {

```

Comment:
Tutaj pomiędzy linijkami jest różnica we wcięciach - można to wyrównać. Poza tym arrow function + let/const

Reply:
Nie wiem skąd wzięły się różnice we wcięciach - w WebStormie wygląda to ok.
Użyłem składni ES5, ponieważ jest to fix do istniejącego skryptu, więc nie ma sensu zaburzać spójności kodu.

Reply:
Czasami zamiast 4 spacji jest tab a jak wiadomo github tabów nie interpretuje jako 4 a jako 8 spacji. Stąd te różnice.
A kod - właśnie nie byłem pewien jakiego standardu używamy w tym pliku dlatego też dałem approve.